src/plugins/shape-generator/index.less

```diff
@@ -20,7 +18,7 @@
 }
 
 /* Hide sliders if the setting is disabled */
-[data-shape-generator-show-sliders="false"]
+.dsm-shape-generator-hide-sliders
```

Comment:
Ah, this shows one downside (upside?) to classes. With datasets, it's easy to get a tristate of (false, true, missing). But it looks like you figured it out.

Reply:
Yup, since the styles only need to be applied to one of those states, a class also works just fine here.